src/main/java/com/iota/iri/crypto/ISS.java

```diff
@@ -87,7 +88,7 @@
     }
 
     /**
-     * Derives a secret key of length {@code FRAGMENT_LENGTH * numberOfFragments} from subseed. <br>
+     * Derives a secret key of length {@code FRAGMENT_LENGTH} * {@code numberOfFragments} from subseed. <br>
```

Comment:
I asked before to change `{@code FRAFMENT_LENGTH}` to `{@value FRAGMENT_LENGTH}` or to `{@link FRAGMENT_LENGTH}`

Any reason why you didn't do that?